torch/testing/_internal/common_modules.py

```diff
@@ -66,7 +73,7 @@ def _parametrize_test(self, test, generic_cls, device_cls):
                 _update_param_kwargs(param_kwargs, 'dtype', dtype)
 
                 try:
-                    active_decorators = []
+                    active_decorators = [set_single_threaded_if_parallel_tbb]
```

Comment:
is it a bad idea to just decorate the gradcheck tests with this instead of putting it here?

Reply:
From reading the surrounding issues around parallel tbb, I think it can also break with the forward pass even though we have not ran across it yet.

Although, maybe all this parallel_tbb code can be removed if the tests are not running anymore? #65092

Reply:
Cool, makes sense. Do you mind adding a comment above this line explaining a bit why the decorator is here? Also, FYI I asked & ParallelTBB will be coming back at some point, so I guess we still need this :/